src/main/java/MultiPaymentExample.java

```diff
@@ -51,11 +51,19 @@ public static void main(String[] args) throws IOException {
 
         // This is where we call our Create method and get our transfer transaction
         // This is where we build our MultiPayment transaction
-        Transaction transaction = new MultiPaymentBuilder()
+        MultiPaymentBuilder multiPaymentBuilder = new MultiPaymentBuilder()
                 .nonce(nonce)
                 .addPayment(recipient1, 100) // amount of arktoshis we want to send
                 .addPayment(recipient2, 200)
-                .addPayment(recipient3, 200)
+                .addPayment(recipient3, 200);
+
+        // This is where we add additional 61 payments
+        for (int i = 0; i < 61; i++) {
+            multiPaymentBuilder.addPayment(recipient1,1);
```

Comment:
- can we change the loop to max=64
- amount can be the value of i

`multiPaymentBuilder.addPayment(recipient1,i); //amount is set in arktoshis`